telemetry_core/src/main/java/com/newrelic/telemetry/json/AttributesJson.java

```diff
@@ -1,36 +1,50 @@
 /*
  * ---------------------------------------------------------------------------------------------
- *  Copyright (c) 2019 New Relic Corporation. All rights reserved.
- *  Licensed under the Apache 2.0 License. See LICENSE in the project root directory for license information.
- * --------------------------------------------------------------------------------------------
+ *   Copyright (c) 2019 New Relic Corporation. All rights reserved.
+ *   Licensed under the Apache 2.0 License. See LICENSE in the project root directory for license information.
+ *  --------------------------------------------------------------------------------------------
  */
 package com.newrelic.telemetry.json;
 
+import java.io.IOException;
+import java.io.StringWriter;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.stream.Collectors;
 
-public interface AttributesJson {
+public class AttributesJson {
 
-  /**
```

Comment:
Oof, isn't it still helpful to have this javadoc here?

Reply:
I don't think anyone outside of us will care about it, because users no longer have to provide it, so no.